### PR TITLE
Add mozorg/about/legal.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9891

### CODE REVIEW

File: en/mozorg/about/legal.ftl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+legal-legal = Legal
+legal-get-involved = Get involved
+legal-protect-the-fox = Protect the Fox
+legal-takedown-requests = Takedown requests
+legal-back-to-legal = Back to Legal
+legal-special-thanks-to-all-of-you = Special thanks to all of you who help report abuses of { -brand-name-mozilla } marks, participate in governance forums, give feedback on our localizations & legal terms, and contribute your skills to the success of the { -brand-name-mozilla } project.
+legal-terms = Terms
+legal-our-websites = Our Websites
+legal-firefox-services = { -brand-name-firefox } Services
```
Comment (peiying2) [inline en/mozorg/about/legal.ftl:15]: Did we miss one entry `Firefox Better Web` which is a product name entry?

File: configs/pontoon.toml [hunk 2]
```diff
@@ -189,6 +189,52 @@ locales = [
 [[paths]]
     reference = "en/mozorg/about/history.ftl"
     l10n = "{locale}/mozorg/about/history.ftl"
+[[paths]]
+    reference = "en/mozorg/about/legal.ftl"
+    l10n = "{locale}/mozorg/about/legal.ftl"
+    locales = [
```
Comment (peiying2) [inline configs/pontoon.toml:195]: Should this be open to all locales rather than those with partial/full localization?

**Review by peiying2 — CHANGES_REQUESTED** [review]
- One product name is missing
- Locale list for this file - should it be open to all?

**Comment by alexgibson** [comment]
Thanks @peiying2 - simplified the config, and also added the missing string (noticed a few more titles that have never been added too).

**Review by peiying2 — CHANGES_REQUESTED** [review]
@alexgibson The config file removes the entry Craig added for his PR yesterday. We need it for the set of privacy files. Other than that, it looks good.

**Comment by alexgibson** [comment]
@peiying2 good catch, updated

**Review by peiying2 — APPROVED** [review]
Thanks for the fix. Looks good now.

**Pull request merged**